docs/cli/how-to-create-model-analysis-cli.md

```diff
@@ -1,157 +1,123 @@
+      target_column_name: ${{inputs.target_column_name}}
+      categorical_column_names: '["Race", "Sex", "Workclass", "Marital Status", "Country", "Occupation"]'
+    outputs:
+      rai_insights_dashboard: ${{outputs.rai_insights_dashboard}}
+
+  explain_01:
+    type: component_job
+    component: azureml:RAIInsightsExplanation:VERSION_REPLACEMENT_STRING
+    inputs:
+      comment: Some random string
+      rai_insights_dashboard: ${{jobs.create-rai-job.outputs.rai_insights_dashboard}}
 
-### Step 5: Submit your job through the CLI
-```
-az ml job create –file pipeline_model_analysis.yaml
-```
+  causal_01:
+    type: component_job
+    component: azureml:RAIInsightsCausal:VERSION_REPLACEMENT_STRING
+    inputs:
+      rai_insights_dashboard: ${{jobs.create-rai-job.outputs.rai_insights_dashboard}}
+      treatment_features: '["Age", "Sex"]'
+      heterogeneity_features: '["Marital Status"]'
 
+  counterfactual_01:
+    type: component_job
+    component: azureml:RAIInsightsCounterfactual:VERSION_REPLACEMENT_STRING
+    inputs:
+      rai_insights_dashboard: ${{jobs.create-rai-job.outputs.rai_insights_dashboard}}
+      total_CFs: 10
+      desired_class: opposite
 
-## Useful CLI Commands
+  error_analysis_01:
+    type: component_job
+    component: azureml:RAIInsightsErrorAnalysis:VERSION_REPLACEMENT_STRING
+    inputs:
+      rai_insights_dashboard: ${{jobs.create-rai-job.outputs.rai_insights_dashboard}}
+      filter_features: '["Race", "Sex", "Workclass", "Marital Status", "Country", "Occupation"]'
 
-Login from CLI:
-```CLI
-az login --tenant <your_tenant_name_such_us_microsoft.onmicrosoft.com>
-```
-Check defaults set:
-```CLI
-az configure
-```
-Set by default Resource Group:
-```CLI
-az configure --defaults group=<your_resource_group_name> location=<your_azure_region>
+  gather_01:
+    type: component_job
+    component: azureml:RAIInsightsGather:VERSION_REPLACEMENT_STRING
+    inputs:
+      constructor: ${{jobs.create-rai-job.outputs.rai_insights_dashboard}}
+      insight_1: ${{jobs.causal_01.outputs.causal}}
+      insight_2: ${{jobs.counterfactual_01.outputs.counterfactual}}
+      insight_3: ${{jobs.error_analysis_01.outputs.error_analysis}}
+      insight_4: ${{jobs.explain_01.outputs.explanation}}
 ```
-Set by default AML Workspace:
-```CLI
-az configure --defaults workspace=<your_workspace_name>
+Save this to a file (e.g. `rai-pipeline.yaml`) and update all of the instances of `VERSION_REPLACEMENT_STRING` with the version number you specified when running `generate_registration_files.py`.
+You will also need to update the `compute:` line to point at the compute resource you wish to use. With those changes made, submit the job using
+```bash
+az ml job create --file ra-pipeline.yaml
 ```
-
-
+The job should appear in the AzureML portal, and you can watch its progress.
+Once complete, go to the 'Models' view in the AzureML portal, and search for the `component_registered_lr_01` model.
```

Comment:
Once complete, go to the 'Models' view (in left hand navigation bar) in AzureML studio to view your Responsible AI dashboard by searching for the `component_registered_lr_01` model, clicking into the model detail and selecting the 'Responsible AI dashboard (preview)' tab.

Reply:
Haven't quite taken this exactly; LMK what you think of the update.